app/models/core.ts

```diff
@@ -79,7 +80,8 @@ export class Graph extends Element {
     exampleTuple = ["hello", 10];
 
     constructor(public plugin: Plugin) {
-        super(plugin.graphPluginData());
+        // TODO: fix that this is obviously special cased
+        super(plugin.graphPluginData("DFAGraph"));
```

Comment:
Remember what it was before? This is the only way to avoid duplication of a bunch of strings. What is your proposal?